app/scripts/lib/metrics.js

```diff
@@ -46,7 +46,7 @@ define([
     'ab'
   ];
 
-  var TEN_MINS_MS = 10 * 60 * 1000;
+  var TWO_MINS_MS = 2 * 60 * 1000;
```

Comment:
Maybe just rename `TWO_MINS_MS` to `DEFAULT_INACTIVITY_TIMEOUT_MS` so that its intent is clearer.